src/main/java/spotify/playback/data/lyrics/GeniusLyricsScraper.java

```diff
@@ -25,6 +29,10 @@ public class GeniusLyricsScraper {
   private static final String USER_AGENT = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36";
   private static final Gson gson = new Gson();
 
+  // Testing fetching higher resolution image - Noah
+  String geniusAlbumArtUrl = "";
```

Comment:
I'm a bit confused why the high-res image stuff is in here. Supposedly to save an extra web request to Genius?

Even still, instance variables like this one should always be marked _private_.